scale/error/models.py

```diff
@@ -59,6 +59,22 @@ def get_nfs_error(self):
         """
         return self.get_builtin_error('nfs')
 
+    def get_invalid_manifest_error(self):
+        """Returns the error for invalid results manifest
+
+                :returns: The invalid results error
+                :rtype: :class:`error.models.Error`
+                """
+        return self.get_builtin_error('invalid-results-manifest')
+
+    def get_missing_output_error(self):
+        """Returns the error for missing a required output
+
+                :returns: The missing output error
+                :rtype: :class:`error.models.Error`
+                """
+        return self.get_builtin_error('missing-required-output')
+
```

Comment:
The new errors should be in the job app fixture file for basic job errors. The fixture files are how these builtin errors get populated in the database. The get_invalid_manifest_error() and get_missing_output_error() methods can be removed. The new job errors can simply be accessed in the job app using get_builtin_error().